.github/workflows/test.yml

```diff
@@ -0,0 +1,12 @@
+name: all test try 1
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install modules
+      run: npm install
+    - name: Run all tests
+      run: npm test
```

Comment:
This will only execute the jest command which will run all the tests however it will not collect coverage information.